README.md

```diff
@@ -10,20 +10,70 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
 
 ![screen shot 2016-04-12 at 12 22 02 pm](https://cloud.githubusercontent.com/assets/4596845/14469165/5d27495c-00b1-11e6-9d28-327938463adf.png)
 
-## Quick start
 
-1. Install Go 1.6, and ensure your `GOPATH` is set. Using [gvm](https://github.com/moovweb/gvm) is recommended.
-1. Install the tool
-
-    ```bash
-    go get github.com/opencontrol/compliance-masonry
-    ```
-
-1. Run the CLI
+## Installation
+Compliance Masonry is packaged into a downloadable executable program for those
+who want to use Compliance Masonry without the need to install any external
+dependencies or programs. This is extremely beneficial for those who may to
+jump right in without worrying about much technical setup.
+
+### Mac OS X Installation
+#### Technical Users - Mac OS X
+In your terminal, run the following:
+```sh
+curl -L https://github.com/opencontrol/compliance-masonry/releases/download/v1.1.2/compliance-masonry_1.1.2_darwin_amd64.zip -o compliance-masonry.zip
```

Comment:
Minor: Have them `cd ~/Downloads` first, _or_ `rm -rf compliance-masonry_1.1.2_darwin_amd64/` after the fact.